### DISCUSSION TITLE
Bind key to arrow icon left/right in leveling track ?

### DISCUSSION THREAD

**Discussion by SebaFR7474** [Lailloken/Exile-UI#381] [Q&A] [answered]
Hi, Is there is way, sorry if it's already implemented, to bind a key to the Arrow icon left/right in Leveling track ?

**Comment by Lailloken** [comment]
Currently not. Are there that many steps you want/have to skip? Exile-leveling has some options to customize the guide-steps:
You can click "build" at the top to disable "league start" (lab trials) and "library" (Siosa quest).

Let me know which steps you're skipping, so I can decide if hotkeys are useful (you're the first to suggest it).

**Comment by SebaFR7474** [comment]
i was doing as a client some carry leveling, since the route is not really the same, it was useful to skip some route.
I made for this one carry leveling an route, it was very long, but if i play with some1 else, who is using an other route, sometimes i prefer to fast forward without needing to stop playing, and i prefer to use a key.

Same when i solo leveling, i go to the Camp of the act i am doing, and i long press on right arrow, to be in the right section, but sometime i dont want to continue to be at the right section in the tip, and want to play, and doing it while i am playing.

And for me clic, on a button instead of with the mouse, is way faster.

And since i have a G502 mouse and even better an Azeron Cyborg, i have a lot of key i can use :)

On other question on the leveling tracking, just up of the left/right arrow we see the act, and it don't often change to the right act i am in, how to change it ?
In the route do we need to put some command, so the leveling tracker now in which act we are ?

Thanks

**Reply by Lailloken** [reply]
Hmm, I'll check if it's possible (I haven't looked at the leveling tracker code in a while). And the settings menu for the leveling tracker is also already really crowded.

> and it don't often change to the right act i am in, how to change it ?

That's for the timer. It shows `total time` | ` act X ` | `act X time`, and the middle part only updates when transitioning between acts while the timer is running.

**Reply by SebaFR7474** [reply]
But how does it know in which act we are ? he often stay to act1 !

**Reply by Lailloken** [reply]
You're supposed to have the timer running from start to finish, without interruption. While it's running, it checks if there's a transition from act X to act X+1. If there is, it saves the act X time in the CSV-file and starts recording the next act's time.

So on a fresh character that just spawned in Twilight Strand, hold the ` < ` button to reset the timer and guide progress, then click the ` act X ` button to start it.

If you want to make the act-counter "catch up" (which doesn't really make sense because that would mean act times have not been recorded correctly), resume the timer and then transition from whichever act it's displaying to the next -- then repeat that until it shows the correct act.

**Reply by SebaFR7474** [reply]
ho okay, i will try that, sometimes for me, and i think i am not the only one, i don"t do all the campain in one time !
Mostly when we are in league start, since it takes 8-14h to do the campaign, when we are not used to do it and not speed runner.

Thanks


What can we do to help you ? thx a lot for you tool, i didn't know and it's very useful ! Need to train for gem info too while leveling and with pob for next season.
and i know i need to understand for all the rest, off leveling tracker.

thanks a lot

**Reply by Lailloken** [reply]
By "without interruption" I meant it has to be running/ticking while you're making progress. Of course you can pause it between sessions or when you're AFK.

**Reply by SebaFR7474** [reply]
Do you think there is a way to stop a timer, and resume it the day after ?

Like we stop it when we want to stop the game.
than later day after, we relaunch it, if we could choose the act where we were, and if it's can  keep the time we stop, the time before.

And then click to resume the timer, something like that ?
It would be awesome !

Do you have a donate button ?

thanks

**Reply by Lailloken** [reply]
You can stop/resume it whenever you want by clicking the ` act # ` button. It also pauses whenever you're tabbed out, or the game isn't running.

I have not been taking donations, but that may change in the future. I certainly appreciate the gesture.

**Comment by Lailloken** [comment, marked as answer]
I'm adding an option to bind the ` < ` ` > ` buttons to hotkeys in the next release because it'll also help with GeForce Now compatibility. Thanks for your suggestion.

**Reply by SebaFR7474** [reply]
That's Awesome thanks alot ! as i use geforce now too sometime, it will help me a lot too :)
